### DISCUSSION TITLE
v2.7.1

### DISCUSSION THREAD

**Discussion by the1812** [the1812/Bilibili-Evolved#4157] [分享 / Share]
✨新增
- `与 "解除 B 站区域限制" 互斥` 现在只在番剧页面生效.
- 更新 `夜间模式` 对新版直播间, 番剧, 稍后再看等页面的适配. (#3963, #4007, #4128, PR #4137 by [Pencil](https://github.com/pencilqaq))
- 新版番剧页面功能适配: `下载视频` (#4044), `自定义顶栏` (#4007)
- 新版稍后再看页面功能适配: `外置稍后再看`, `启用快速收藏`, `播放器控制栏背景`, `展开视频简介`. (#3916)
- `简化直播间 - 付费礼物` 现在对底栏只留一个包裹按钮, 因为其他元素也都算付费礼物了.

🐛修复
- 修复 `自定义顶栏` 动态的详情链接. (PR #4119 by [星海](https://github.com/lovelyCARDINAL))
- 修复在部分分辨率下 `外置稍后再看` 和 `启用快速收藏` 的响应式宽度. (#4083)
- 修复 `直播全屏包裹` 和 `简化直播间 - 付费礼物` 在新版直播间下的适配问题. (#4105, #4108, PR #4122 by [Pencil](https://github.com/pencilqaq), PR #4123 by [Pencil](https://github.com/pencilqaq))
- 修复 `自定义顶栏` 未完全隐藏个人空间原版顶栏. (#4079)
- 修复 `直播信息扩充` 在新版动态无法打开 “正在直播” 十个以后的链接. (#4034, #3917, PR #4145 by [hhp1614](https://github.com/hhp1614))

☕开发者相关
- pnpm 升级至 8.1.0, 之前本地装过依赖的开发者需要重新 `pnpm install` 一下.
- 废弃 Ajax Hook API. (#4117)
- 新增 API `addVideoActionButton`, 可向视频操作按钮区的 "收藏" 右侧添加元素.

<hr /><em>This discussion was created from the release <a href='https://github.com/the1812/Bilibili-Evolved/releases/tag/v2.7.1'>v2.7.1</a>.</em>

**Comment by kethoven** [comment]
适配新版稍后再看页面。能不能扩大到适配所有的https://www.bilibili.com/list/* 页面。
包括收藏夹播放，稍后再看，up主全部视频播放页面。这些页面应该都是相同的问题。

**Reply by the1812** [reply]
加了
GitHub 源: v2.7.1-50-gdd29a9806
jsDelivr 源: v2.7.1-54-gabadc5e0f

**Reply by alex402566** [reply]
下载的2.7.1在油猴安装怎么是2.5.2版本？

**Reply by Geeyun-JY3** [reply]
> 下载的2.7.1在油猴安装怎么是2.5.2版本？

下载什么？不会是下了 release 附的压缩包吧？
安装方法看 README：https://github.com/the1812/Bilibili-Evolved#%E5%AE%89%E8%A3%85

**Comment by AzusaHana** [comment]
与 "解除 B 站区域限制" 互斥
当前仍然在 如 https://www.bilibili.com/bangumi/media/md28412513 互斥
希望能排除 https://www.bilibili.com/bangumi/media/ 这个页面

**Reply by Tinhone** [reply]
可以查看 `设置>兼容性选项` 并截图反馈

**Reply by AzusaHana** [reply]
![image](https://github.com/the1812/Bilibili-Evolved/assets/100906636/6f4a1d61-d3a5-4f33-b5b5-bed0e883c2b1)
![image](https://github.com/the1812/Bilibili-Evolved/assets/100906636/8db7195d-3067-43af-9234-67a232e7f458)
Commit: 0ba5f2f11
内部名称: compatibilities

**Reply by AzusaHana** [reply]
> 可以查看 `设置>兼容性选项` 并截图反馈

![image](https://github.com/the1812/Bilibili-Evolved/assets/100906636/89e269d2-bf92-455e-b14f-74c211222d6b)
这是关掉的

**Comment by Nyte-BK201** [comment]
早期版本的自定义顶栏支持自定义顶栏颜色。现在这个功能已经无了吗？
我最开始就是因为顶栏太难用，才开始用bili-evolved。感谢老哥维护了这么多年，b站的网页端更新就像他们的KPI一样，改这么勤快也不会提升使用体验，每次更新插件都要修。不容易。

**Reply by Geeyun-JY3** [reply]
自定义顶栏的颜色就是通用设置里的主题颜色啊

**Reply by Nyte-BK201** [reply]
谢了，我以为主题色是固定颜色。不过因为b站改版，改了颜色没有白色好看，还是没动他了。 :)
